apps/docs/src/common/page-wrap.tsx

```diff
@@ -18,7 +18,6 @@ export const PageWrap: FC<PageProps> = ({ children }) => {
       footerContent="Copyright (C) 2020, 2021 Crown Copyright"
       navigation={navigation}
       meta={[
-        { href: '/accessibility-statement', text: 'Accessibility statement' },
```

Comment:
Is there a reason this has been removed?

Reply:
Forgot to add it back in after testing which page-wrap document to use. Have done so now.